### DISCUSSION TITLE
Default Id pattern doesn't support A-Z anymore

### DISCUSSION THREAD

**Discussion by MedLabs** [pocketbase/pocketbase#6183] [Q&A]
Hi,
I updated my DB to 0.23 and I got some problems adding new data then I found that the new ID pattern does not support A-Z in the 15 chars string...
Don't you think that including A-Z with a-z and 0-9 could generate more unique Ids ? why A-Z was removed from the default pattern ?
I added A-Z manually to my id models... but I think, removing it should be manually.
It's just an opinion

**Comment by ganigeorgiev** [comment]
See https://github.com/pocketbase/pocketbase/discussions/6170#discussioncomment-11653594.
